issue_07/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+
+    <img id="js-loadingImg" src="loading-circle.gif" alt="ローディング画像">
```

Comment:
Try running it in JS instead of HTML. Create a function for the loading part.

Reply:
function to create a loading section in JS.
However, I am not too sure about what to define within the function.
https://github.com/yoshikawa-akr/js-lessons/pull/7/commits/ac48f08bd01ad521093f3527ca7a501c69b10d58